### ISSUE TITLE
[Panic] Brief description of the issue

### ISSUE THREAD

**Issue by SeamanDog** [Raytwo/Cobalt#53]
### System Firmware Version

19.0.1

### Atmosphere Version

1.8.0

### Cobalt Version

1.16.0

### Steps to reproduce

Booting Fire Emblem Engage with Wi-Fi enabled causes Cobalt to panic. Booting in Airplane mode fixes the panic and lets the game load normally with Cobalt and mods functioning.

### Details

Error Code: 2--0069

Cobalt v1.16.0
Location: M:\.cargo\git\checkouts\rust-native-tls-e1c2f7824b5e2733\c40ba64\src\imp\nnsdk.rs:192:9

not implemented

**Comment by oncceh** [comment]
Same bug

**Comment by Autliety** [comment]
same issue reported on Cobalt v1.23.0

**Comment by Raytwo** [comment]
Sounds like you're using Yuzu or a derivate, which as indicated on the installation page, is not supported

**Comment by seaman-271** [comment]
> Sounds like you're using Yuzu or a derivate, which as indicated on the installation page, is not supported


No, this occurs on real hardware, which I was using. Its an issue with some wifi connections liking the auto updater, which can't be turned off. Which is why I said I have to turn on airplane mode.
